user-guide/Advanced_Functionality/Databases/OpenSearch_database.md

```diff
@@ -16,23 +16,297 @@ Supported versions:
+opensearch.ssl.verificationMode: certificate
+
+# If opensearch.ssl.verificationMode is full or certificate, specify the full path to one or more CA certificates that comprise a trusted chain for your OpenSearch cluster. For example, you might need to include a root CA and an intermediate CA if you used the intermediate CA to issue your admin, client, and node certificates.
+
+# Below is an example of the rootCA.crt certificate that was generated using the generate-tls-certificates from the Skyline public GitHub repository. See also "Update the trusted root certificates with rootCA.crt" topic further up in this help file.
+opensearch.ssl.certificateAuthorities: ["/usr/local/share/ca-certificates/rootCA.crt"]
+
+
+# If your OpenSearch is protected with basic authentication(using the /etc/opensearch/opensearch-security/internal_users.yml), these settings provide the username and password that the OpenSearch Dashboards server uses to connect to OpenSearch.
+# More information can be found here: https://opensearch.org/docs/latest/install-and-configure/install-opensearch/debian/#configure-a-user and https://opensearch.org/docs/latest/security/authentication-backends/authc-index/#authentication-flow
+opensearch.username: userName
+opensearch.password: pwd
+```
+
+See also the [official documentation](https://opensearch.org/docs/latest/) on how to set up your OpenSearch cluster in case something isn't covered.
+
+> [!NOTE]
+> Contrary to Elasticsearch, OpenSearch is only supported on Linux and not on Windows.
+
 ## Connecting your DMS to an OpenSearch cluster
 
 To configure the connection to an OpenSearch database, configure the settings as detailed in [Cassandra database](xref:Configuring_the_database_settings_in_Cube#cassandra-database).
 
 > [!IMPORTANT]
 > An OpenSearch database requires a separate Cassandra cluster or Amazon Keyspaces.
+
+## Troubleshooting
+
+### Remote certificate invalid
+
+```text
+System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.
+```
+
+Above exception in the SLDBConnection logging can be fixed by adding the rootCA.crt in the  *Trusted Root Certification Authorities* store.
+
+### Multiple DNS names for IP
+
+In case multiple DNS names point to a single IP-address, set the below option to false in the opensearch.yml file:
+
+```yaml
+plugins.security.ssl.transport.resolve_hostname: false
+```
+
+See also [(Advanced) Hostname verification and DNS lookup](https://opensearch.org/docs/1.2/security-plugin/configuration/tls/#advanced-hostname-verification-and-dns-lookup).
```

Comment:
We propably want to add the 'transport is no longer supported' error in here. The root cause of this error is the nodes_dn not matching the certificates subject, but the error is super vague and does not help investigating the issue.